Octokit.Tests/Clients/DeploymentsClientTests.cs

```diff
@@ -45,24 +50,44 @@ public void RequestsCorrectUrl()
         {
             var connection = Substitute.For<IApiConnection>();
             var client = new DeploymentsClient(connection);
-            var expectedUrl = "repos/owner/name/deployments";
+            var expectedUrl = ApiUrls.Deployments(owner, name);
+
+            client.GetAll(owner, name);
+            connection.Received(1)
+                .GetAll<Deployment>(Arg.Is<Uri>(u => u == expectedUrl), Args.ApiOptions);
+        }
 
-            client.GetAll("owner", "name");
-            connection.Received(1).GetAll<Deployment>(Arg.Is<Uri>(u => u.ToString() == expectedUrl));
+        [Fact]
+        public void RequestsCorrectUrlWithApiOptions()
+        {
+            var connection = Substitute.For<IApiConnection>();
+            var client = new DeploymentsClient(connection);
+            var expectedUrl = ApiUrls.Deployments(owner, name);
+
+            var options = new ApiOptions
+            {
+                PageSize = 1,
+                PageCount = 1,
+                StartPage = 1
+            };
+
+            client.GetAll(owner, name, options);
+            connection.Received(1)
+                .GetAll<Deployment>(Arg.Is<Uri>(u => u == expectedUrl), options);
         }
     }
 
     public class TheCreateMethod
     {
-        readonly NewDeployment newDeployment = new NewDeployment("aRef");
+        private readonly NewDeployment _newDeployment = new NewDeployment("aRef");
```

Comment:
I'm not a fan of using `_` or `m_` prefixes for private fields like this. Could you change this back?